extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/SparkRangerAdminPlugin.scala

```diff
@@ -76,4 +94,38 @@ object SparkRangerAdminPlugin extends RangerBasePlugin("spark", "sparkSql") {
     val digits = s"regexp_replace($lower, '[0-9]', 'n'$pos)"
     digits
   }
+
+  @throws[AccessControlException]
+  def verify(
+      requests: util.List[RangerAccessRequest],
```

Comment:
import java.util.List

Reply:
When with imported ` java.util.List`, a format warning is shown by IDEA. Does it matter?
<img width="418" alt="image" src="https://user-images.githubusercontent.com/1935105/187893801-6b5b6674-af83-4fe4-b210-3dbd3fb9f6c9.png">

If autofix by IDEA with prefix, falling back to `util.List`.

Reply:
`util.List` is good, to distinguish w/ scala List

Reply:
OK. let's keep it with util prefix.